.travis.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - stage: post-tests
       name: code coverage
       os: osx
-      osx_image: xcode10.2
+      osx_image: xcode11.3
```

Comment:
it seems `llvm-cov` does not have the lcov output format as of LLVM/Xcode 10. I bumped this and went ahead and bumped all of our other Xcode versions on Travis too.